Content.Client/SS220/SpaceWars/Party/UI/CustomControls/ReceivedPartyInvitePanel.xaml.cs

```diff
@@ -0,0 +1,76 @@
+
+using Content.Client.SS220.UserInterface.Utility;
+using Content.Shared.SS220.SpaceWars.Party;
+using Robust.Client.AutoGenerated;
+using Robust.Client.Graphics;
+using Robust.Client.Player;
+using Robust.Client.UserInterface.Controls;
+using Robust.Client.UserInterface.XAML;
+using Robust.Shared.Utility;
+
+namespace Content.Client.SS220.SpaceWars.Party.UI.CustomControls;
+
+[GenerateTypedNameReferences]
+public sealed partial class ReceivedPartyInvitePanel : PanelContainer
+{
+    [Dependency] private readonly IPartyManager _party = default!;
+    [Dependency] private readonly IPlayerManager _player = default!;
+
+    public readonly IPartyInvite Invite;
+    private readonly QuickConfirmationWindow _confirmationWindow = new();
+
+    public ReceivedPartyInvitePanel(IPartyInvite invite)
+    {
+        RobustXamlLoader.Load(this);
+        IoCManager.InjectDependencies(this);
+
+        DebugTools.Assert(invite.Kind is PartyInviteKind.Received);
+        DebugTools.AssertEqual(invite.Target, _player.LocalUser);
+        Invite = invite;
+
+        _party.InviteUpdated += args =>
+        {
+            if (args != Invite)
+                return;
+
+            Refresh();
+        };
+
+        AcceptButton.OnPressed += _ => Accept();
+        DenyButton.OnPressed += _ => Deny();
+
+        PanelOverride = new StyleBoxFlat { BackgroundColor = new Color(60, 60, 60) };
```

Comment:
Где-то я уже видел этот цвет...
Лучше стоит создать static class с цветами для этой фичи, чтобы всё выглядело единообразно и легко изменялось
```suggestion
        PanelOverride = new StyleBoxFlat { BackgroundColor = DefinetlyNotTheRandomColor };
```